assets/src/scss/layout/_navbar.scss

```diff
@@ -261,34 +341,33 @@ a.nav-link {
     transform: scaleX(0);
     transition: transform 0.25s;
     z-index: -1;
+
+    @include large-and-up {
+      border-bottom-width: 4px;
+    }
   }
 }
 
 .active a.nav-link {
   --nav-link-active-- {
-    color: var(--nav-link--color, $white);
-    text-decoration-color: $gp-green;
-  }
+    color: $white;
 
-  opacity: 1;
-  text-decoration-line: underline;
-  text-decoration-thickness: 3px;
-  text-underline-offset: auto;
-  text-underline-position: under;
```

Comment:
Those styles were there for the burger menu, the underlining of the links should stay static and have the length of the text (that's different from the desktop or mobile tabs menus), so we need a different underline system in the burger menu. I kept this style and used those variables for the animated underline so there's only one type of variable to use for both (color, thickness), but you can probably find a better way.
The goal at the time was to have an underline like this (just the length of the text)
![Screenshot from 2022-03-01 10-05-18](https://user-images.githubusercontent.com/617346/156138758-2ec28e76-5559-461a-89d5-4a35fead00bd.png)
and a link that's clickable on the major part of the width of the menu, you can check that on [international](https://www-dev.greenpeace.org/international/act/) for example
(unless the design team decided otherwise recently, of course :grimacing:)

Reply:
Yes! I about about to answer that 😄 since It's currently in draft ⚒️